oss_src/unity/python/sframe/cython/cy_sarray.pyx

```diff
@@ -429,3 +429,9 @@ cdef class UnitySArrayProxy:
         with nogil:
             proxy = (self.thisptr.copy_range(start, step, end))
         return create_proxy_wrapper_from_existing_proxy(self._cli, proxy)
+    
+    cpdef rolling_apply(self, string fn_name, ssize_t before, ssize_t after, size_t min_observations):
```

Comment:
I think you should call this builtin_rolling_apply because, in the future, we will probably have a pure rolling_apply?